<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCalc">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calc</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbTime">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOur">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>timestamp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>value</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
